src/main/java/frc/robot/subsystems/Intake/IntakeSubsystem.java

```diff
@@ -65,7 +64,7 @@ public void periodic() {
 
     pidOutput = MathUtil.clamp(pidOutput, -1, 1);
 
-    intakeIO.setArmSpeed(pidOutput);
+    intakeIO.setArmSpeed(0);
```

Comment:
Could you add a comment here we disable the intake arm motor because the mount for the intake arm is falling off.